src/templates/project.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+
+export default ({ data }) => {
+  const project = data.markdownRemark
+  return (
+    <div>
+      <h1>
+        {project.frontmatter.title}
+      </h1>
+      <div dangerouslySetInnerHTML={{ __html: project.html }} />
```

Comment:
If i'm understanding this correctly we're grabbing raw html from a graphql object and passing it in?

Could you elaborate a bit as to how this works?

Reply:
Sure! Here is the explanation from the Gatsby doc:
>  Source plugins bring data into Gatsby’s data system and transformer plugins transform raw content brought by source plugins. This simple pattern can handle all data sourcing and data transformation you might need when building a Gatsby site.

So the transformer plugin we are using is "gatbsy-transformer-remark". As stated in their docs: 
https://www.gatsbyjs.org/packages/gatsby-transformer-remark/
> This plugin adds additional fields to the MarkdownRemark GraphQL type including html, excerpt, headers, etc. Other Gatsby plugins can also add additional fields.

Here's a screenshot of the graphiql query/response:

![screen shot 2017-09-28 at 11 07 40 am](https://user-images.githubusercontent.com/5697474/30977725-0ac05168-a43e-11e7-8b63-ac348acbb5ba.png)